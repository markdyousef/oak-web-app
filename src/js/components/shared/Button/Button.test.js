// @flow
import React from 'react';
import renderer from 'react-test-renderer';
import { SquareButton } from './index';

test('renders SquareButton', () => {
    const props = {
        onClick: () => {},
        text: 'Hello'
    };
    const component = renderer.create(
        <SquareButton {...props} />
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});
