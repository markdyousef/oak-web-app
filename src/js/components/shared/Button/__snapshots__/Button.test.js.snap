// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders SquareButton 1`] = `
<button
  className="hzsDdV"
  onClick={[Function]}
  type={undefined}
>
  Hello
</button>
`;
